tests/integration_python/test_run_cli.py

```diff
@@ -918,24 +918,24 @@ def test_task_environment(
     }
 
     manifest_content["feature"] = {
-        "010": {"dependencies": {"package": "==0.1.0"}},
-        "020": {"dependencies": {"package": "==0.2.0"}},
+        "010": {"dependencies": {"package2": "==0.1.0"}},
+        "020": {"dependencies": {"package2": "==0.2.0"}},
     }
 
-    manifest_content["environments"] = {"env_010": ["010"], "env_020": ["020"]}
+    manifest_content["environments"] = {"env-010": ["010"], "env-020": ["020"]}
 
     manifest_content["tasks"] = {
-        "task1": "package",
+        "task1": "cat $(which package2)",
```

Comment:
Is there a reason why you changed that?

`deno_task_shell` doesn't contain `which` so this will fail on Windows: https://pixi.sh/latest/workspace/advanced_tasks/#built-in-commands